apps/ui/src/style.scss

```diff
@@ -304,7 +304,7 @@ input:placeholder-shown {
   .s-label:has(+ textarea),
   .s-base:has(textarea:only-child) {
      &::before {
-      @apply block bg-gradient-to-b from-skin-border to-transparent h-2 w-full absolute top-[20px] left-0;
+      @apply block bg-gradient-to-b from-skin-border to-transparent h-[8px] w-full absolute top-3.5 left-0;
```

Comment:
Not sure why `h-2` was changed to `h-[8px]`.

Reply:
For consistency, everything that is not spacing related (`(m|p|space|gap)-[0-9]`) is using arbitrary pixels value.

we're already using `h-[32px]` for avatar, etc ...

Or should we keep using both h-2 and h-[8px] ?

Reply:
I see, let's keep it like this then.